### DISCUSSION TITLE
UUID v7 in .NET 9

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#111] [General]
# UUID v7 in .NET 9

.NET natively knows UUID v4 aka Guid.NewGuid(). But with preview 7 we get Version 7 of UUID's natively.

https://steven-giesel.com/blogPost/ea42a518-4d8b-4e08-8f73-e542bdd3b983

**Comment by sonergonul** [comment]
How can I use this CreateVersion7 method? Downloaded .NET 9 preview 6 but no luck.

**Reply by linkdotnet** [reply]
This will come with preview 7 - so somewhen august
